modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/change/tracking/spi/reference/DDMFormInstanceRecordVersionTableReferenceDefinition.java

```diff
@@ -29,6 +31,15 @@ public class DDMFormInstanceRecordVersionTableReferenceDefinition
 	public void defineChildTableReferences(
 		ChildTableReferenceInfoBuilder<DDMFormInstanceRecordVersionTable>
 			childTableReferenceInfoBuilder) {
+
+		childTableReferenceInfoBuilder.singleColumnReference(
+			DDMFormInstanceRecordVersionTable.INSTANCE.
+				formInstanceRecordVersionId,
+			AssetEntryTable.INSTANCE.classPK
+		).singleColumnReference(
```

Comment:
Since AssetEntry has both classNameId and classPK, we should use classNameReference. Otherwise, we could get references from other entities with the same classPK.